hmpo-lev/email/messages/messages_en.properties

```diff
@@ -0,0 +1,42 @@
+emailVerificationSubject=Welcome to LEV
+emailVerificationLinkTextHtml=Verify your email address
+passwordResetSubject=Reset your LEV password
+passwordResetLinkTextHtml=Reset your LEV password
```

Comment:
Do these `...LinkTextHtml` values actually get used anywhere?

Reply:
Not any more - removed